ws-nextjs-app/pages/[service]/live/[id]/Header/styles.tsx

```diff
@@ -42,12 +48,31 @@ export default {
         width: '100%',
       },
     }),
-  textContainerMediaOpen: ({ mq }: Theme) =>
+  LiveMediaBaseStyling: ({ mq, spacings }: Theme) =>
```

Comment:
For consistency
```suggestion
  liveMediaBaseStyling: ({ mq, spacings }: Theme) =>
```

Reply:
Maybe we could just call it liveMedia?
```suggestion
  liveMedia: ({ mq, spacings }: Theme) =>
```